assets/dev/js/editor/elements/views/base.js

```diff
@@ -226,9 +226,31 @@ BaseElementView = BaseContainer.extend( {
 
 	getHandlesOverlay: function() {
 		const $handlesOverlay = jQuery( '<div>', { class: 'elementor-element-overlay' } ),
-			$overlayList = jQuery( '<ul>', { class: `elementor-editor-element-settings elementor-editor-${ this.getElementType() }-settings` } );
+			$overlayList = jQuery( '<ul>', { class: `elementor-editor-element-settings elementor-editor-${ this.getElementType() }-settings` } ),
+			elementType = this.options.model.get( 'elType' ),
```

Comment:
It seems like the method you use in the line above (`this.getElementType()`) gives you the `elementType`